web/src/ButtonLink.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import styled from "styled-components"
+import { Button } from "./Button"
+
+let ButtonLinkRoot = styled.a`
+  ${Button}
+`
+
+type ButtonLinkProps = {
+  href: string
+  label: string
+  target?: string
+  rel?: string
+}
+
+function ButtonLink(props: ButtonLinkProps) {
+  return (
+    <ButtonLinkRoot href={props.href} {...props}>
```

Comment:
the {...props} here feels like a bug waiting to happen, can we just list out target and rel explicitly?